packages/babel-plugin-transform-react-jsx-source/test/fixtures/react-source/basic-sample/expected.js

```diff
@@ -1,4 +1,5 @@
+var __jsxFileName = "/I/am/not/sure/how/to/get/path/to/test/fixtures/react-source/basic-sample/actual.js";
```

Comment:
Any suggestions on how can I avoid putting absolute path here for tests? Avoid using babel-helper-plugin-test-runner and have something custom or patching testRunner to support fuzzy matches?